rt-forecast-2/forecast/update-rt-forecast.R

```diff
@@ -0,0 +1,76 @@
+# Packages -----------------------------------------------------------------
+require(EpiNow2)
+require(data.table)
+require(future)
+
+# Define a target date ----------------------------------------------------
+
+# target_date <- Sys.Date()
+
+
+# Update delays -----------------------------------------------------------
+
+generation_time <- readRDS(here::here("rt-forecast-2", "forecast", "delays", "data", "generation_time.rds"))
+incubation_period <- readRDS(here::here("rt-forecast-2","forecast", "delays", "data", "incubation_period.rds"))
+reporting_delay <- readRDS(here::here("rt-forecast-2","forecast", "delays", "data", "onset_to_death_delay.rds"))
+
+
+# Get deaths  ---------------------------------------------------------------
+source(here::here("utils", "get-us-data.R"))
+
+# Get raw data
+deaths_raw <- get_us_deaths(data = "daily")
+
+# Format for Epinow
+deaths_national <- deaths_raw %>%
+  dplyr::group_by(date) %>%
+  dplyr::summarise(deaths = sum(deaths), .groups = "drop_last") %>%
+  dplyr::rename(confirm = deaths) %>%
+  dplyr::mutate(region = "US")
+
+deaths <- deaths_raw %>%
+  dplyr::rename(confirm = deaths, region = state) %>%
+  dplyr::select(date, region, confirm) %>%
+  dplyr::bind_rows(deaths_national)
+
+# Format for epinow2 ------------------------------------------------------
+
+deaths <- setDT(deaths)
+deaths <- deaths[, .SD[date >= (max(date) - lubridate::weeks(6))], by = region]
```

Comment:
I used 6 weeks based on nothing really :) - in the global one I moved to 8 weeks and that might be sensible here. I also added a burn_in arg cut the early estimates that don't have any data (set that to 7 days again based on nothing).